deps/spidershim/src/v8arraybuffer.cc

```diff
@@ -122,6 +122,14 @@ ArrayBufferView::ByteLength()
   return JS_GetDataViewByteLength(view);
 }
 
+void
+ArrayBuffer::Neuter()
+{
+  JSContext* cx = JSContextFromIsolate(Isolate::GetCurrent());
+  JS::RootedObject obj(cx, &reinterpret_cast<JS::Value*>(this)->toObject());
+  JS_DetachArrayBuffer(cx, obj, KeepData);
```

Comment:
Hmm, what happens to the underlying buffer at this stage?  As far as I can tell, we'd be leaking it here...